src/components/messenger/list/index.tsx

```diff
@@ -155,7 +155,18 @@ export class Container extends React.Component<Properties, State> {
     return users.map((user) => ({ ...user, image: user.profileImage }));
   };
 
+  isUserAlreadyInConversation = (userId: string) => {
+    return this.props.conversations
+      .filter((conversation) => conversation.isOneOnOne === true)
```

Comment:
```suggestion
      .filter((c) => c.isOneOnOne)
```

`isOneOnOne` is already a boolean value.